src/test/java/org/folio/edge/rtac/RtacHandlerTest.java

```diff
@@ -0,0 +1,145 @@
+package org.folio.edge.rtac;
+
+import static org.apache.http.HttpStatus.SC_NOT_ACCEPTABLE;
+import static org.apache.http.HttpStatus.SC_OK;
+import static org.folio.edge.core.Constants.APPLICATION_JSON;
+import static org.folio.edge.core.Constants.APPLICATION_XML;
+import static org.folio.edge.core.Constants.TEXT_PLAIN;
+import static org.junit.Assert.assertEquals;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.restassured.RestAssured;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.stream.Collectors;
+import org.apache.commons.lang3.ArrayUtils;
+import org.apache.http.HttpHeaders;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.edge.rtac.model.Instances;
+import org.folio.edge.rtac.utils.RtacMockOkapi;
+import org.junit.Test;
+
+public class RtacHandlerTest extends MainVerticleTest {
+
+  private static final Logger logger = LogManager.getLogger(RtacHandlerTest.class);
+
+  private String prepareQueryFor(String apiKey, String... instanceIds) {
+    if (ArrayUtils.isEmpty(instanceIds)) {
+      return String.format("/rtac?apikey=%s", apiKey);
+    } else {
+      String instancesAsString = Arrays.asList(instanceIds).stream()
+          .collect(Collectors.joining(","));
+      return String.format("/rtac?apikey=%s&instanceIds=%s", apiKey, instancesAsString);
+    }
+  }
+
+  private Instances prepareRecordsFor(String... instanceIds) {
+    if (ArrayUtils.isEmpty(instanceIds)) {
+      throw new IllegalArgumentException("No instances specified");
+
+    } else {
+      final var holdings = Arrays.asList(instanceIds).stream().map(RtacMockOkapi::getHoldings)
+          .collect(Collectors.toList());
+      final var instanceHoldingRecords = new Instances();
+      instanceHoldingRecords.setHoldings(holdings);
+      return instanceHoldingRecords;
+    }
+  }
+
+  @Test
+  public void checkNoAcceptHeadersSpecifiedSuccessfulResponse() throws IOException {
+    logger.info("=== Test when No Accept headers specified returns response in XML by default ===");
```

Comment:
What information does this log message provide that the name of the test does not?

Reply:
These log messaged were in the test class initially, so I just leaved them as it was. But possibly you are right - they are redundant here and just provide some king of test's separation in the log file. I'll remove them, to minimize tests output. Thank you.

Reply:
Oh ok, I thought the test class was new in this pull request

Reply:
Anyway, the latest code base has been cleaned from these extra logs, in this way there will be not produced long logs ).